packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -344,7 +342,13 @@ class Uploader {
       resume: true,
       retryDelays: [0, 1000, 3000, 5000],
       uploadSize: this.bytesWritten,
-      metadata,
+      metadata: Object.assign(
+        {
+          // file name and type as specified by tus protocol
```

Comment:
Nitpicking, but is it defined by the protocol, or just the tusd implementation? I didn’t find it in the protocol, so maybe linking here: https://github.com/tus/tusd/blob/5b376141903c1fd64480c06dde3dfe61d191e53d/unrouted_handler.go#L614-L646?

Reply:
`filename` is mentioned a few times in protocol https://github.com/tus/tus-resumable-upload-protocol/blob/master/protocol.md#protocol-extensions

Reply:
since `filetype` is not documented, I have added a comment to the URL you shared